apps/api/urls.py

```diff
@@ -10,16 +10,18 @@
 router.register(r"sessions", views.ExperimentSessionViewSet, basename="session")
 
 connect_patterns = [
-    path("generate_key/", views.generate_key, name="generate_key"),
-    path("callback/", views.callback, name="callback"),
-    path("consent/", views.consent, name="consent"),
+    path("generate_key", views.generate_key, name="generate_key"),
+    path("callback", views.callback, name="callback"),
+    path("consent", views.consent, name="consent"),
 ]
 
 urlpatterns = [
     path("participants/", views.update_participant_data, name="update-participant-data"),
+    # Duplicate update-participant-data without a trailing "/" for backwards compatibility
+    path("participants", views.update_participant_data, name="update-participant-data"),
```

Comment:
I confirmed that when we run `reverse("api:update-participant-data")` it returns the route without the slash